app/src/main/java/com/gatchimath/totask/Task.java

```diff
@@ -3,73 +3,73 @@
 /**
  * Basic task class.
  *
- * @author  gatchi (github.com/gatchi)
```

Comment:
These changes were produced by IntelliJ's auto-formatter. I've reverted this file to avoid changes to unrelated files.